src/grpc/lightning.ts

```diff
@@ -289,23 +289,24 @@ export function keysend(
   sphinxLogger.info('keysend', logging.Lightning)
   return new Promise(async function (resolve, reject) {
     try {
-      const randoStr = crypto.randomBytes(32).toString('hex')
-      const preimage = ByteBuffer.fromHex(randoStr)
+      const preimage = crypto.randomBytes(32)
+      const payment_hash = sha.sha256.arrayBuffer(preimage)
       const dest_custom_records = {
         [`${LND_KEYSEND_KEY}`]: preimage,
       }
       if (opts.extra_tlv) {
         Object.entries(opts.extra_tlv).forEach(([k, v]) => {
-          dest_custom_records[k] = ByteBuffer.fromUTF8(v)
+          dest_custom_records[k] = Buffer.from(v)
```

Comment:
I'm confused why did we change `fromUTF8` to just `from`

Reply:
why was here ByteBuffer in the first place?

Reply:
anyway i dont see why bytebuffer should be used here, its an external (3rd party) library, nodejs has a native buffer and grpc uses the nodejs native buffer